frontends/web/src/components/fullscreen/fullscreen.tsx

```diff
@@ -16,19 +16,30 @@
 
 import { h, RenderableProps } from 'preact';
 import { AppLogo } from '../icon';
+import { Footer } from '../layout';
+import { SwissMadeOpenSource } from '../icon/logo';
 import * as style from './fullscreen.css';
 
 interface Props {
+    withBottomBar?: boolean;
```

Comment:
But I can't find where this prop is set. Is it in another commit?

Reply:
same commit in `frontends/web/src/routes/device/bitbox02/bitbox02.tsx`